appveyor.yml

```diff
@@ -22,7 +22,7 @@ install:
   - "python -c \"import struct; print(struct.calcsize('P') * 8)\""
 
   # Install the dependencies of the project (skip nibabel for speed)
-  - "conda install --yes --quiet setuptools numpy scipy matplotlib scikit-learn nose mayavi pandas h5py PIL patsy"
+  - "conda install --yes --quiet setuptools numpy scipy matplotlib scikit-learn nose mayavi pandas h5py PIL patsy pyside"
```

Comment:
>
> Oh sorry I think I misunderstood you, you mean skip the test if the backend is not available, right?
>
> yes detect is the working backend is not present and if not return a
meaningful error message.

then we can test for 1 travis build that things don't crash if backend is
not there but works accross versions is available.

ok?